server/routes/azure/azure_routes.py

```
@@ -3,20 +3,21 @@
 from utils.logging.secure_logging import mask_credential_value
-from utils.auth.token_management import get_token_data, store_tokens_in_db  # Using same util for DB storage
+from utils.auth.token_management import get_token_data
 import json
 
 azure_bp = Blueprint("azure_bp", __name__)
 
 # ---- Azure Routes ------------------------------------------------------#
 @azure_bp.route("/azure/login", methods=["POST", "GET", "OPTIONS"])
-def azure_login_route():
+@require_permission("connectors", "write")
+def azure_login_route(user_id):
```

Ruff (0.15.2)

[warning] Unused function argument: `user_id`

(ARG001)